teeworlds-server-mod/teeworlds-server-mod.json

```diff
@@ -0,0 +1,78 @@
+{
+  "pufferd": {
+    "type": "teeworlds",
+    "display": "Teeworlds Mod Server",
+    "install": {	
+      "commands": [
+		{
+		  "type": "download",
+		  "files": [
+			"${url}"
+		  ]
+		},
+		{
+          "type": "writefile",
```

Comment:
Is there really a reason to write a file with the args? Why does this need a start.sh file?

Reply:
Yes it is, as the executable can be any value and vars dont work in here...

Reply:
Then, it seems like the binary name needs to use that variable too? You have it currently hardcoded to the DD name in the start.sh you write.

Reply:
Thx, committed a fix for that.